src/index.html

```diff
@@ -1,17 +1,200 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
+    <meta name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form action="http://localhost:8080/" 
+    method="post" 
+    autocomplete="off">
+      <fieldset 
+      class="group"
+      >
```

Comment:
Check the code style everywhere. BTW in this case you can write it in one line
```suggestion
      <fieldset 
        class="group"
      >
```

or 

```suggestion
      <fieldset class="group">
```